app/src/main/java/com/teamagam/gimelgimel/app/network/receivers/NetworkChangeReceiver.java

```diff
@@ -29,9 +29,14 @@
 
     @Override
     public void onReceive(Context context, Intent intent) {
-        boolean isNetworkAvailable = isNetworkAvailable(context);
+        try {
```

Comment:
Well, if the code will throw an exception it wont be caught anywhere and crash the app.
It will still do the same here, but atleast it'll log it.
I'll remove the re-throw line.